pkg/quota/v1/resources.go

```diff
@@ -47,21 +47,7 @@ func Equals(a corev1.ResourceList, b corev1.ResourceList) bool {
 
 // V1Equals returns true if the two lists are equivalent
 func V1Equals(a corev1.ResourceList, b corev1.ResourceList) bool {
```

Comment:
Should we eliminate redundant functions here?
 `func Equals(a corev1.ResourceList, b corev1.ResourceList) bool`  and `func V1Equals(a corev1.ResourceList, b corev1.ResourceList) bool`.

Bye the way, should we use `reflect.DeepEqual()` as the function implementation?

Reply:
Eliminating `V1Equals` might like backward incompatible change (controllers can depend on this package, right?). But changing in-tree callers of `V1Equals` (I believe there is only one) to just call `Equals` instead might be reasonable.

Reply:
Besides, the performance of `reflect.DeepEqual()` is much worse than the current implementation.
/assign @liggitt would you mind having a look.

Reply:
go ahead and drop the V1Equals function, and update the single caller to call Equals

I agree we should not switch to using DeepEqual internally

Reply:
`V1Equals` has been removed in the latest pull request. @liggitt